.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - iojs-v2.0.0
-  - iojs
+  - "4.0"
+  - "4.1"
```

Comment:
In that case you'd want `node`, since travis uses `nvm`'s syntax. (unless "latest" is some magical travis thing that proxies to `nvm`). Kind of sad to explicitly drop support for node 4 tho, I'm not sure I see the benefit of dropping it or the cost of keeping it.